.eslintrc.js

```diff
@@ -21,6 +21,7 @@ module.exports = {
   plugins: ['prettier', 'json', 'jsx-a11y'],
   rules: {
     'react/jsx-one-expression-per-line': 'off',
+    'import/no-cycle': 'off',
```

Comment:
The new version of `eslint` produces 10x errors of the form:
```
/path/to/simorgh/src/app/containers/Article/index.jsx
  10:1  error  Dependency cycle via ../Text:6=>../Paragraph:2=>../InlineLink:5=>../../routes:10  import/no-cycle
```

This is a temporary measure to unblock other work/fix the vuln. Will open separate issue to properly fix and remove this rule.